Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/CrowdStrikeFalcon.py

```diff
@@ -2735,6 +2783,27 @@ def search_device_by_ip(raw_res, ip_address):
     return raw_res
 
 
+def enrich_groups(all_group_ids) -> Dict[str, Any]:
+    """
+        Receives a list of group_ids
+        Returns a dict {group_id: group_name}
+    """
+    result = dict()
+    groups_list = list()
+    for group in range(0, len(all_group_ids), 500):
+        groups_list.append(all_group_ids[group:group + 500])
+    for ids in groups_list:
+        params = {'ids': ids}
+        response_json = http_request('GET', '/devices/entities/host-groups/v1', params, status_code=404)
+        try:
+            for resource in response_json['resources']:
+                item = {resource['id']: resource['name']}
+                result.update(item)
+        except KeyError:
+            demisto.debug("Some groups were not found")
+    return result
```

Comment:
following the note in the previous review, this is what I meant:
```suggestion
    result = dict()
    params = {'ids': all_group_ids}
    response_json = http_request('GET', '/devices/entities/host-groups/v1', params, status_code=404)
    for resource in response_json['resources']:
        try:
            result[resource['id']] = resource['name']
        except KeyError:
            demisto.debug(f"Could not retrieve group name for {resource=}")
    return result
```

Reply:
Done and testing!